myapp/spec/helpers/sessions_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the SessionsHelper. For example:
+#
+# describe SessionsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe SessionsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Aren't you going to write tests for the session helper?

Reply:
I am adding some more helpers in step 18 and 19, and started writing test for them, so it will come in next PR.